core/tests/test_utils.py

```diff
@@ -168,6 +197,9 @@ def log_line(self, line):
         print '%s%s' % (LOG_LINE_PREFIX, line)
 
     def _delete_all_models(self):
+        """Deletes all keys from the NDB datastore. Subclasses of TestBase
```

Comment:
At this level we shouldn't refer to NDB since TestBase is supposed to be platform-independent. So, strike "NDB".